### ISSUE TITLE
 The loss value was always at 87.3365. 

### ISSUE THREAD

**Issue by AmmyXu** [D-X-Y/Auto-ReID-and-Others#14]
Hello, D-X-Y.
Thank you very much for sharing. It helps me a lot.
However, when I was training on your data set, I discovered that the loss value was always at 87.3365. I tried several networks. This is the case. What could be the cause? How to modify it?

**Comment by susu3621** [comment]
I get the same issue. With caffenet.
The loss value is not decressing, and stay at  87.3365.

Iteration 7820 / 18000 (5.46453 iter/s, 3.65997s/20 iters), loss = 131.005      [ 0:31 (H:M) ]
I0419 16:44:32.729363 46286 solver.cpp:245]     Train net output #0: loss = 87.3365 (* 1 = 87.3365 loss)
I0419 16:44:32.729374 46286 solver.cpp:245]     Train net output #1: loss_dif = 87.3365 (* 0.5 = 43.6683 loss)
I0419 16:44:32.729382 46286 sgd_solver.cpp:105] Iteration 7820, lr = 0.001

**Comment by AmmyXu** [comment]
set lr = 0.0005

**Comment by susu3621** [comment]
Thanks Ammy! lr = 0.0005 works.
